BlueprintUI/Sources/Internal/OSLog+Blueprint.swift

```diff
@@ -12,6 +12,20 @@ extension OSLog {
 /// Namespace for logging config.
 public enum BlueprintLogging {
 
+    /// Configuration for logging options
+    public struct Config {
+
+        /// When `true`, measurement data will be collected for all `Element`s
+        public var recordMeasurePasses: Bool
```

Comment:
Yeah, "measure pass" is kind of vague in this context. Enum is OK, or if you want a fine-grained config I'd call this something like `recordElementMeasures`.